src/main/java/dk/kb/ds/cumulus/export/CalendarUtils.java

```diff
@@ -145,6 +146,23 @@ private static String parseYearMonthDay(String datetime) {
         log.trace("parseYearMonthDay({}) failed parsing", datetime);
         return null;
     }
+
+    private final static Pattern YEAR_TO_YEAR = Pattern.compile("^([0-9]{4}).([0-9]{4})$");
+    /**
+     * Parses inputs consisting of exactly 4 digits (year), a non-digit character, 4 digits (year).
+     * @param datetime year-year, e.g. '2019-2020'.
+     * @return Truncated ISO-8601 representation of the given datetime if possible, else null.
+     */
+    private static String parseYearToYear(String datetime) {
+        Matcher yyMatcher = YEAR_TO_YEAR.matcher(datetime);
+        if (yyMatcher.matches()) {
+            return yyMatcher.group(1)
+                + "-" + yyMatcher.group(2);
```

Comment:
Expressing ranges for the Solr `DateRangeField` is described at https://lucene.apache.org/solr/guide/8_1/working-with-dates.html#date-range-formatting for year to year it follows the pattern `[1987 TO 2001]'.

Reply:
Ah, yes and I even knew that.